### PR TITLE
Add a flag to keep the custom ttl header

### PR DESCRIPTION
**Pull request by alexander-schranz**
Allows to keep the custom ttl header for additional caches.

### CODE REVIEW

File: src/SymfonyCache/CustomTtlListener.php [hunk 1]
```diff
@@ -38,11 +43,13 @@ class CustomTtlListener implements EventSubscriberInterface
     public const SMAXAGE_BACKUP = 'FOS-Smaxage-Backup';
 
     /**
-     * @param string $ttlHeader The header that is used to specify the TTL header
+     * @param string $ttlHeader     The header that is used to specify the TTL header
```
Comment (dbu) [inline src/SymfonyCache/CustomTtlListener.php:46]:
```suggestion
     * @param string $ttlHeader     The header name that is used to specify the time to live
```

File: src/SymfonyCache/CustomTtlListener.php [hunk 2]
```diff
@@ -38,11 +43,13 @@ class CustomTtlListener implements EventSubscriberInterface
...
+     * @param bool   $keepTtlHeader Keep the custom TTL header for later usage
```
Comment (dbu) [inline src/SymfonyCache/CustomTtlListener.php:47]:
```suggestion
     * @param bool   $keepTtlHeader Keep the custom TTL header on the response for later usage (e.g. debugging)
```

**Review by dbu — COMMENTED** [review]
thanks, makes sense to me. can you please add a changelog entry?

i also suggested an edit to the (existing) ttlHeader phpdoc because it was a bit weird.

**Comment by dbu** [comment]
thanks. released as https://github.com/FriendsOfSymfony/FOSHttpCache/releases/tag/2.11.0

**Comment by alexander-schranz** [comment]
@dbu oh you were fast. Thx for creating the changelog entry :)

**Pull request merged**